solutions/CF Jury Marks.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: jury-marks
+source: CF
+title: Jury Marks
+author: Danh Ta Chi Thanh
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/53302)
+
+<LanguageSection>
+
+<CPPSection>
+
+**Time Complexity:** $\mathcal{O}(K^2 \log K)$
+
+## C++ Implementation
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios_base::sync_with_stdio(0);
+	cin.tie(0);
+	int K, N;
+	cin >> K >> N;
+	vector<int> a, b;
+	a.resize(K+1);
+	for (int i = 1; i <= K; ++i) {
+		cin >> a[i];
+		a[i] += a[i-1];
+	}
+	b.resize(N); 
+	for (int& p : b) cin >> p;
+	set<int> init;
+	for (int m = 1; m <= K; ++m) {
+		init.insert(b.front() - a[m]);
+	}
+	int ans = 0;
+	for (const int& d : init) { //must be `const int` (or `auto` for simplicity)

```

Comment:
or just `int d: init`?

Reply:
I used `int d : init` and it got a compilation error.

Reply:
> or just `int d: init`?

https://codeforces.com/contest/831/submission/107320598

Reply:
https://ideone.com/q1HAIz

Reply:
I've just changed as you said and it seemed fine. 
https://codeforces.com/contest/831/submission/107377082